src/PKSim.Presentation/Views/IResizableWithDefaultHeightView.cs

```diff
@@ -0,0 +1,13 @@
+﻿using OSPSuite.Presentation.Views;
+
+namespace PKSim.Presentation.Views
+{
+   public interface IResizableWithDefaultHeightView : IResizableView
+   {
+      /// <summary>
+      /// Default height placeholder that will be used to initialize the container of this view
+      /// This is required to avoid a flickering effect. 
+      /// </summary>
+      int DefaultHeight { get; }
```

Comment:
This is new. I am giving a default height so that we do not have the flickering effect on BUILDING The view